website/docs/automation/gitops/argocd/app-of-apps/index.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Argo CD App of Apps"
+chapter: true
+sidebar_position: 100
+---
+
+[Argo CD](https://argoproj.github.io/cd/) can deploy a set of applications to different environments (DEV, TEST, PROD ...) using common set of Kubernetes manifest and customizations specific to an environment.
+
+We can leverage [Argo CD App of Apps pattern](https://argo-cd.readthedocs.io/en/stable/operator-manual/cluster-bootstrapping/) to implement this use case. This pattern allows us to specify one Argo CD Application that consists of other applications.
+
+![argo-cd-app-of-apps](assets/argocd-app-of-apps.png)
+
+We reference [EKS Workshop Git repository](https://github.com/aws-samples/eks-workshop-v2/tree/main/environment/workspace/manifests) as a Git repository with Common Manifest. This repository will contain an initial resource state for each application.
```

Comment:
Unsure what "Common Manifest" means or why its capitalized

Reply:
Changed to the terms used in the workshop introduction: "base" manifests